verilog/analysis/verilog_linter_test.cc

```diff
@@ -439,6 +441,87 @@ TEST(VerilogLinterDocumentationTest, AllRulesMarkdown) {
   EXPECT_TRUE(absl::StrContains(stream.str(), "Enabled by default:"));
 }
 
+TEST(VerilogLinterDocumentationTest, PrintLintRuleFile) {
+  auto config_status = verilog::LinterConfigurationFromFlags("");
+  EXPECT_TRUE(config_status.ok());
+  if (!config_status.ok()) {
+    return;
+  }
+  const LinterConfiguration &config = *config_status;
+
+  // Generate Line Rule File
+  std::ostringstream stream;
+  verilog::GetLintRuleFile(&stream, config);
+
+  // Spot-check a few patterns, must mostly make sure generation
+  // works without any fatal errors.
+  // NOTE: This will break if/when the rules change so this part
+  // of the test is not ideal.
+  EXPECT_TRUE(absl::StrContains(stream.str(), "always-comb"));
+  EXPECT_TRUE(absl::StrContains(
+      stream.str(), "module-filename=allow-dash-for-underscore:false"));
+  EXPECT_TRUE(absl::StrContains(stream.str(), "-forbid-negative-array-dim"));
+
```

Comment:
If you move the `const std::string generated_default_rules_str = stream.str();` further up, directly after the `GetLintRuleFile()` call, you can use the same readable name in the other tests above as well, where `stream.str()` is used.

Reply:
Yep, agree.